### ISSUE TITLE
Why is "add_question_sql" storing vectors as question+sql? SQL isn't natural language, so it wouldn't affect the semantic matching of input queries?

### ISSUE THREAD

**Issue by qingwu11** [vanna-ai/vanna#447]
![ques+sql](https://github.com/vanna-ai/vanna/assets/41281228/86408651-cbd9-4cdd-bf2f-210b1e182dd8)
Why is "add_question_sql" storing vectors as question+sql? SQL isn't natural language, so it wouldn't affect the semantic matching of input queries?

**Comment by erickrribeiro** [comment]
Yes, @qingwu11. I also wondered the same thing when I saw how this id is created. I think that only the question field should be used to generate the unique code and not the question+sql composition

**Comment by zainhoda** [comment]
In our testing we actually did find better search by embedding the SQL query particularly if it contains specific terminology that may not be contained in the question.

However, to @erickrribeiro 's point -- I think that the ID should only be based on the question
